student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(id, name = 'Unknown', age = 0, parent_permission: true, classroom: nil)
+    super(id, name, age, parent_permission)
```

Comment:
- [ ] Kindly remove the `id`. As we refactored earlier in the Person class which Student inherits from `id` is auto-generated.

- [ ] The `age` and `classroom` are not an optional parameters so they should be declared correctly.

- [ ] The `parent_permission` should be assigned correctly in the `super` method call.

  ```
  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    ...
  ```